configs/default/APEX-APEXF7.config

```diff
@@ -0,0 +1,105 @@
+# Betaflight / STM32F7X2 (S7X2) 4.1.6 Apr 25 2020 / 05:12:26 (283bda8bf) MSP API: 1.42
+# manufacturer_id: APEX   board_name: APEXF7   custom defaults: NO
```

Comment:
Please remove this line.

Reply:
done :)

Reply:
Now it is missing the `# Betaflight...` line - this one needs to stay in, the `# manufacturer_id` line needs to go.

Reply:
sorry, just realized that.